src/kernel/include/shared/cos_types.h

```diff
@@ -187,6 +187,9 @@ typedef enum {
 
 #define CAPTBL_EXPAND_SZ 128
 
+#define COS_VIRT_MACH_COUNT        2
```

Comment:
`VM` instead of `VIRT_MACH`?

This is very specific to your component.  I don't think it should be in `cos_types.h` which is for system-wide customization.